pyiron_atomistics/calphy/job.py

```diff
@@ -651,34 +721,82 @@ def collect_general_output(self):
             None
         """
         if self._data is not None:
+            #solid liquid specific outputs
             if "spring_constant" in self._data["average"].keys():
                 self.output.spring_constant = self._data["average"]["spring_constant"]
+            if "density" in self._data["average"].keys():
+                self.output.equilibrium_density = self._data["average"]["density"]
+            self.output.equilibrium_volume_atom = self._data["average"]["vol_atom"]
+
+
+            #main results from mode fe
+            self.output.temperature = self.input.temperature
+            self.output.pressure = self.input.pressure
             self.output.energy_free = self._data["results"]["free_energy"]
             self.output.energy_free_error = self._data["results"]["error"]
             self.output.energy_free_reference = self._data["results"][
                 "reference_system"
             ]
             self.output.energy_work = self._data["results"]["work"]
-            self.output.temperature = self.input.temperature
+            self.output.energy_pressure = self._data["results"]["pv"]
+
+            #collect ediffs and so on
             f_ediff, b_ediff, flambda, blambda = self.collect_ediff()
             self.output.forward_energy_diff = list(f_ediff)
             self.output.backward_energy_diff = list(b_ediff)
             self.output.forward_lambda = list(flambda)
             self.output.backward_lambda = list(blambda)
+
+            #get final structure
+            traj = Trajectory(os.path.join(self.working_directory, "conf.equilibration.dump"))
+            aseobj = traj[0].to_ase(species=self.calc.element)[0]
```

Comment:
What's the last `[0]` do?

Reply:
Also why the conversion `pyiron.Atoms` -> `ase.Atoms` and then back again, when `aseobj` doesn't seem to be used afterwards?

Reply:
[`Trajectory`](https://pyscal.org/en/latest/examples/15_trajectory_module.html) is from pyscal to handle LAMMPS dump files. It always returns a list, therefore the last `[0]` in line 752. `Trajectory` reads dump file, returns ASE atoms, which I convert to pyiron atoms and is stored.

Reply:
Ah, my bad.  I sort of assumed it's our Trajectory and got confused.  Can you maybe rename it in the import?

Reply:
Renamed as PyscalTrajectory